cmd/kpromo/cmd/pr/pr.go

```diff
@@ -158,6 +158,8 @@ func init() {
 	}
 }
 
+// Note: whoever made this, it's too complex and hard to read for human and machine,
```

Comment:
please drop this note, or rewrite it in a better way

Reply:
sorry about writing note that because its complexity, I was wondering 🤔after seeing a commit history

![image](https://github.com/kubernetes-sigs/promo-tools/assets/17626300/17f2bb76-8e56-41bf-bc3f-12cf058164cd)

how that's function are functionally when complexity reached 20+

Reply:
I've rewrite it in a better way